tests/ra_aid/test_llm.py

```
@@ -51,6 +52,16 @@ def test_initialize_expert_openai_custom(clean_env, mock_openai, monkeypatch):
         model="gpt-4-preview"
     )
 
+def test_initialize_expert_gemini(clean_env, mock_gemini, monkeypatch):
+    """Test expert Gemini initialization."""
+    monkeypatch.setenv("EXPERT_GEMINI_API_KEY", "test-key")
+    llm = initialize_expert_llm("gemini", "gemini-2.0-flash-thinking-exp-1219")
```

Ruff (0.8.2)

Local variable `llm` is assigned to but never used

Remove assignment to unused variable `llm`

(F841)